jenkins/Jenkinsfile.premerge

```diff
@@ -72,3 +78,24 @@ pipeline {
         }
     } // end of stages
 } // end of pipeline
+
+boolean isSignedOff() {
+    def target_rev = sh(returnStdout: true,
+            script: "git rev-parse refs/remotes/origin/${ghprbTargetBranch}").trim()
+    def revs_arr = sh(returnStdout: true,
+            script: "git log ${target_rev}..${ghprbActualCommit} --pretty=format:%h").split()
+
+    def signed_off = false
+    for( String commit : revs_arr ) {
+        def signed_log = sh(returnStdout: true,
+                script: "git show ${commit} --shortstat | grep 'Signed-off-by'")
```

Comment:
So does this mean that if I do one commit with no sign off, and then I do another commit with one that this will pass?